tutorials/gf/CMakeLists.txt

```diff
@@ -0,0 +1,28 @@
+# This is a minimal CMake file that illustrates how to use Eigen
+# in project that uses ALPSCore
+
+# we need at least cmake version 3.1
+cmake_minimum_required(VERSION 3.1)
+
+# the project is called 'use_eigen', and it is a C++ project
+project(gf CXX)
+
+set(CMAKE_CXX_STANDARD 11)
+
+# the project relies on the ALPSCore package. If ALPSCore is not found
+# automatically, specify its location using:
+# export ALPSCore_DIR=/location/to/ALPSCORE/
+find_package(ALPSCore 2.2 REQUIRED gf)
+
+# check if Eigen is present:
```

Comment:
This check (5 lines) is not needed: Eigen is not used by the tutorial code.
```suggestion

```

Reply:
Eigen is used by tensors, that are the container for GFs

Reply:
But it is the same. GF is almost header only, and it will include the Eigen.
And if ALPSCore does not provide Eigen I suppose it's not gonna work then.

Reply:
No, it's not the same:

Situation 1, pertinent to Eigen-in-ALPSCore tutorial: Suppose some future version of ALPSCore will not use Eigen in utilities. I built ALPSCore without Eigen, but my program uses Eigen. I want to know whether i need to request it.

Situation 2, pertinent to GF tutorial: Suppose some future version of ALPSCore will not use Eigen even in GF. I built ALPSCore and want to use GF. Even if ALPSCore does not provide Eigen, i do not need to request it my code does not need it.

So, the difference is "my code uses Eigen directly" vs. "my code uses GF, which happens to use Eigen in its current implementation".